tests/beaker_tests/cisco_bgp_neighbor_af/test_bgpneighboraf.rb

```diff
@@ -529,7 +592,9 @@ def build_manifest_bgp_nbr_af(tests, id)
                tests[id][:title_pattern])
   tests[id][:manifest] = "cat <<EOF >#{PUPPETMASTER_MANIFESTPATH}
   node 'default' {
-    cisco_bgp_neighbor_af { '#{tests[id][:title_pattern]}':
+    #{extra_config}
+
+    cisco_bgp_neighbor_af { '#{af[:asn]} #{af[:vrf]} #{af[:neighbor]} #{af[:afi]} #{af[:safi]}':
```

Comment:
This is needed change due to "neighbor" parameter in Section 3. Test Pattern Testing.
@glennmatthews